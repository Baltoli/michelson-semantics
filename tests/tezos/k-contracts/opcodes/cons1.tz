$contract { parameter int;
storage (list int);
code { UNPAIR; CONS; NIL operation; PAIR; };
 } ;
$param -5 ;
$storage { 10 } ;
$ExpectedOutput { -5 ; 10 } ;

