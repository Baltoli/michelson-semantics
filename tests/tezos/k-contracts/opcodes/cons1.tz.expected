<generatedTop>
  <k>
    .
  </k>
  <stack>
    Pair .List ListItem ( -5 )
    ListItem ( 10 )
  </stack>
  <paramtype>
    int .AnnotationList
  </paramtype>
  <storagetype>
    list .AnnotationList int .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
